_blog/2023-12-29-using-loco-rust-rails.mdx

```diff
@@ -0,0 +1,433 @@
+In this article we're going to talk about how you can get started with Loco - a new web framework that builds on Axum. We will cover getting started using controllers, migrations, middleware, static files and handling a database.
+
+TODO
+
+## Getting Started
+
+To get started, you will need to make sure to have Loco's CLI installed by using the following:
+
+```bash
+
```

Comment:
All codeblocks have superfluous surrounding blank lines, and extra blank lines between each line

Reply:
It looks like trying to C+P from a new tool I've been trying out straight into Vim wasn't a good idea :facepalm: